main.go

```diff
@@ -49,15 +53,22 @@ var filterZones = FlagArray(
 	`Explicitly forward just several DNS zones. A zone of "example.com" matches "example.com" and all of its subdomains. (separated by comma - "ipinfo.io,ipify.org",)`,
 )
 
-// DNS suffix that will use the bypass proxy. A zone of
-// //
 func main() {
 	flag.Parse()
 
 	dialerUpstreamURL, err := url.Parse(*proxyUpstreamURL)
 	if err != nil || dialerUpstreamURL.Scheme != "http" {
 		log.Fatalf("Invalid upstream URL: %s", *proxyUpstreamURL)
 	}
+
+	sm, err := proxy.NewStickyMapper(*stickyStorage)
+	if err != nil {
+		log.Fatalf("Failed to create sticky mapper, %v", err)
+	}
+
+	api := api.NewServer(*proxyAPIAddr, sm.Save)
```

Comment:
It would be more clear if you pass StickyMapper interface where needed instead of abstract func because when reading code it more clear there it comes from and no other changes needed to signature if you need more methods from StickyMapper later.

Reply:
Passing interface now.